tests/application/static_site/test_views.py

```diff
@@ -407,7 +407,7 @@ def test_view_sandbox_topic(test_app_client, logged_in_rdu_user):
 
 def test_measure_page_social_sharing(app, test_app_client, logged_in_rdu_user):
```

Comment:
We probably want a test that checks that `description` is used for social sharing if one is provided?

Reply:
Good idea, done. 👍